conanfile.py

```diff
@@ -84,6 +80,3 @@ def package_info(self):
         bindir = os.path.join(self.package_folder, "bin")
         self.output.info("Appending PATH environment variable: {}".format(bindir))
         self.env_info.PATH.append(bindir)
-
-        # Don't update gclient automatically when running it
-        self.env_info.DEPOT_TOOLS_UPDATE = "0"
```

Comment:
This concerns me

Doesn't it mean that we always execute the latest version, no matter which version we explicitly require, e.g. `depot_tools/20200515`?

Reply:
Yes, I don't like that either. Unfortunately, there's an issue in the depot_tools bootstrapping scripts on Windows. I.e. if `DEPOT_TOOLS_UPDATE` is set to zero, it doesn't run the bootstrapping properly and fails to run.

See:
* https://github.com/bincrafters/community/issues/943#issuecomment-629307967
* https://bugs.chromium.org/p/chromium/issues/detail?id=1067590#c7

Reply:
Frankly, I feel depot_tools doesn't really play well with conan's "reproducible build philosophy". The before-mentioned bootstrapping must always be run on the first invocation of many depot_tools tools...

As far as I can see, this will pretty much inevitably mean that depot_tools will fiddle around with the package content on first invocation.

Reply:
This means effectively that we are adding a `latest` version, something we said we don't do again 

I understand that having finally Python 3 support is important so we might do an exception here. At least if this is temporary and upstream plans to fix it

At the very least, we need a new issue in the Bincrafters' issue tracker with the information, and an in-source comment pointing to it so we won't forget about it

Reply:
Sounds reasonable. Could you open an issue? I'll adapt the PR accordingly tomorrow morning.

Reply:
Nevermind. Done. :)